.github/workflows/update-nightlies.yaml

```diff
@@ -61,8 +61,22 @@ jobs:
         for x in ${{ matrix.files }}; do
           curl https://storage.googleapis.com/knative-nightly/${{ matrix.nightly }}/latest/$x > ${GITHUB_WORKSPACE}/${{ matrix.directory }}/$x
         done
+
+        # Prior to the cut we have output in the following format:
+        #
+        # - serving.knative.dev/release: "v20210111-9f0302e4"
+        # + serving.knative.dev/release: "v20210112-9f0302e4"
+        #
+        unique_refs=$(git diff | grep serving.knative.dev/release | tr -s ' ' | cut -c43-50 | sort -u | wc -l)
```

Comment:
This works because this is just serving, which on deeper consideration may be ok.  We should monitor this closely to ensure it keeps working.

Reply:
yeah - that's why the guard is `unique-refs != 1`. Thus if it's `0` then the PR will go through.